libcaf_io/src/basp_broker.cpp

```diff
@@ -121,6 +121,7 @@ behavior basp_broker::make_behavior() {
     send(this, tick_atom::value, heartbeat_interval);
   }
   return {
+
```

Comment:
O'rly?

Reply:
This newline only kept `clang-format` from formatting the lambdas below aligned after the `{`. I did find a better solution though: `behavior{` is a tiny bit more verbose but it gets `clang-format` to correctly align the lambdas two-spaces from the left.